src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,660 @@
+report to your site's (i.e. the first-party's) reporting service, but that's a decision
+that the framed site would have to make.
+
+{% Aside 'gotchas' %}
+
+In Chrome DevTools, you'll see a console error or warning pop up for violations that are
+committed by third-party script but also by third-party iframes.
+
+{% endAside %}
+
+## Browser support
+
+The table below sums up browser support for the **Reporting API v1**, that is with the
+`Reporting-Endpoints` header. Browser support for the legacy Reporting API (`Report-To`
+header) is the same, except for one report type: Network Error Logging isn't supported in
+the new Reporting API. Read the [migration
+guide](/reporting-api-migration/#network-error-logging) for details.
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Report type</th>
+        <th>Chrome</th>
+        <th>Chrome iOS</th>
+        <th>Safari</th>
+        <th>Firefox</th>
+        <th>Edge</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>CSP violation (Level 3 only)</td>
+        <td>✔ Yes</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✔ Yes</td>
+      </tr>
+      <tr>
+        <td>Network Error Logging</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+      </tr>
+      <tr>
+        <td>All other: COOP/COEP violation, Document Policy violation, Deprecation, Intervention, Crash</td>
+        <td>✔ Yes</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✘ No</td>
+        <td>✔ Yes</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Related reporting mechanisms
+
+### Report-Only
+
+CSP, COOP and COEP headers each have a `Report-Only` variant:
+[`Content-Security-Policy-Report-Only`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only),
+[`Cross-Origin-Embedder-Policy-Report-Only`](https://web.dev/coop-coep/#group) and
+[`Cross-Origin-Opener-Policy-Report-Only`](https://web.dev/coop-coep/#group). Both the
+main headers and their `Report-Only` variant are configured using `report-to`. Once
+`report-to` is configured, you need to specify endpoints.
+
+Do so by using the Reporting API headers⏤preferably, the new header `Reporting-Endpoints`
+(Reporting API v1) instead of the legacy one `Report-To` (v0). CSP has its own
+specificities with `report-uri`. See details in the [migration
+guide](/reporting-api-migration).
+
+{% Aside %} One example where `...-Report-Only` headers are useful: let's assume you have
+a CSP set up for your site. You plan on tightening up your site's security by making your
+CSP stricter. Instead of directly deploying this new, stricter CSP, you first deploy it in
+`Report-Only` mode. This helps you monitor what your CSP _could have broken_ and tweak it,
+instead of actually inducing breakage for your end-users. {% endAside %}
+
+### ReportingObserver
```

Comment:
```suggestion
### `ReportingObserver`
```

Reply:
Cool TIL, I thought `` couldn't be used in headings.